Cross Region VM Restore Points.md

```diff
@@ -0,0 +1,396 @@
+                  {
+                     "path": "Path-Where-To-Place-Public-Key-On-VM",
+                     "keyData": "PEM-Encoded-public-key-file"
+                  }
+               ]
+            }
+         }
+      },
+      "storageProfile": {
+         "osDisk": {
+            "osType": "Windows|Linux",
+            "name": "OSDiskName",
+            "diskSizeGB": "10",
+            "caching": "ReadWrite",
+            "managedDisk": {
+               "id": "CSM Id of the managed disk",
+               "storageAccountType": "Standard_LRS"
+            },
+            "diskRestorePoint": {
+                   "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>",
+   }
+         },
+         "dataDisks": [
+            {
+               "lun": "0",
+               "name": "datadisk0",
+               "diskSizeGB": "10",
+               "caching": "ReadWrite",
+               "managedDisk": {
+                  "id": "CSM Id of the managed disk",
+                  "storageAccountType": "Standard_LRS"
+               },
+               "diskRestorePoint": {
+                   "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>",
+               }
+            }
+         ]
+      },
+      "diagnosticsProfile": {
+         "bootDiagnostics": {
+            "enabled": true,
+            "storageUri": " http://storageaccount.blob.core.windows.net/"
+         }
+      },
+   },
+   "provisioningState": "Succeeded | Failed | Creating | Deleting",
+   "provisioningDetails": {
+      "creationTime": "Creation Time of Restore point in UTC",
+   },
+   "instanceView": {
+      "statuses": [{
+          "code": "ReplicationState/succeeded",
+          "level": "Info",
+          "displayStatus": "Replication succeeded",
+          "message": "<will be populated in error scenarios>",
+          "time": "2019-10-14T21:29:47.477089+00:00"
+      }],
+      "diskRestorePoints": [{
+          "id": "<diskRestorePoint Arm Id>",
+  “replicationStatus” : {
+"status": {
+          	"code": "ReplicationState/succeeded",
+          	"level": "Info",
+          	"displayStatus": "Replication succeeded",
+          	"message": "<will be populated in error scenarios>",
+          	"time": "2019-10-14T21:29:47.477089+00:00"
+          	}
+  “completionPercent” : <completion percentage of the replication>
+}
+      }],
+    }  
+}
+```
+
+## Known issues and limitations
+* Cross-region copy functionality is only supported via REST APIs. ARM Template deployment, CLI, SDK, PS will be added later. 
+* Copy of copy does not work - You cannot copy a restore point that is already copied from another region. For ex. if you have copied a RP1 from East US to West US as RRP1. You cannot copy RRP1 from West US to another region (or back to East US). 
```

Comment:
Interesting - is this a private preview constraint or is this expected behavior for GA?

Reply:
Its a private preview limitation. This will be fixed.